src/Service.GraphQLBuilder/Sql/SchemaConverter.cs

```diff
@@ -250,5 +235,36 @@ public static string GetGraphQLTypeForColumnType(Type type)
                         subStatusCode: DataApiBuilderException.SubStatusCodes.GraphQLMapping)
             };
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="metadataValue"></param>
+        /// <returns></returns>
+        /// <exception cref="DataApiBuilderException"></exception>
+        public static IValueNode CreateValueNodeFromDbObjectMetadata(object metadataValue)
+        {
+            IValueNode arg = metadataValue switch
```

Comment:
what if metadataValue is of type guid?

Reply:
Updated handling for GUID. I noted in a different comment: this method is a helper function that wasn't modified for use in this pr besides breaking out into helper function. (i originally thought I would re-use it elsewhere).